### ISSUE TITLE
Air 33.1.1.217, Windows (DirectX 11) crash with the error Process finished with exit code -1073741819 (0xC0000005)

### ISSUE THREAD

**Issue by denisgl7** [Gamua/Adobe-Runtime-Support#436]
It looks like solving the memory leak issues created new
With version 33.1.1.217 Windows application crashes with an error - Process finished with exit code -1073741819 (0xC0000005)
Starling is loading, sprites with few objects too. But it costs a little load and the application crashes.
With the previous SDK, this problem is not.
I can also add the line
-swf-version=37
And the problem disappears. That is, the problem is in the implementation DirectX 11

Has anyone tested their Windows applications with a new SDK?

**Comment by aclipski** [comment]
Yes, this is the same thing we're experiencing in our game:

https://github.com/Gamua/Adobe-Runtime-Support/issues/20#issuecomment-675795420

**Comment by atmofunk** [comment]
I, too, am having this error with .217 + .259
.98 is fine.

aclipski's issue looks to be my issue as well, but in my case i'm using Starling, not Away3D.   From my Event Viewer:

Faulting module path: C:\WINDOWS\system32\d3d11.dll

**Comment by ajwfrost** [comment]
Yes - known issue (and we have another bug open for it too), we have a fix that will be in the next release. We can provide an updated DLL to patch this if you wanted it urgently?

**Comment by denisgl7** [comment]
I would try with a new DLL

**Comment by aclipski** [comment]
Yes, an updated DLL would be fantastic!

**Comment by ajwfrost** [comment]
This is available from the below link:
https://transfer.harman.com/messages/sN8r93jkimvNhr0IslQxXP/attachments/HefTUKNKVRnc3bxudQauxF/download/Adobe%20AIR.dll
Checksum (SHA256): 8454acbb53ac5ae69e134ea4b71944040727061f2dfe63333cfcadf7a35b7503

**Comment by aclipski** [comment]
@ajwfrost Sorry, I'm just now getting around to trying this out. I'm assuming this goes into runtimes/air. It looks like this is a 32-bit DLL so it would go into the win folder? 

Our game runs in 64-bit on Windows, so would there be a way to get a 64-bit version of this DLL as well?

**Comment by ajwfrost** [comment]
@aclipski  try this one:
https://transfer.harman.com/messages/zZmqbVDi7gaxnVsge4nxo5/attachments/dBRd3GObSy80ffyVZ3tYT4/download/Adobe%20AIR.dll
Checksum (SHA256): d6e24de4df5ff3cb8c4e4bfcf8cdbfbdffd75d8358d2975106def17c56a9a1bf 

thanks

**Comment by aclipski** [comment]
This seems to have fixed the problem, thank you!

**Comment by ll24llll** [comment]
I'm facing this with AIR33.1.1.259, Window 10 Pro v1903

**Comment by ajwfrost** [comment]
FYI should be fixed in the 33.1.1.300 release..
